ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts

```diff
@@ -0,0 +1,193 @@
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (small)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Wallop Large',
+      type: 'StartsUsing',
+      netRegex: { id: '8E82', source: 'Ihuykatumu Ivy', capture: false },
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (big)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Sneeze',
+      type: 'StartsUsing',
+      netRegex: { id: '8E7B', source: 'Drowsie', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Ihuykatumu Drowsie Flagrant Spread',
+      type: 'HeadMarker',
+      netRegex: { id: '008B' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.moveAway(),
+    },
+
+    // ** Apollyon ** //
+    {
+      id: 'Ihuykatumu Apollyon Blade',
+      type: 'StartsUsing',
+      netRegex: { id: ['8DFB', '8E04'], source: 'Apollyon' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Ihuykatumu Apollyon High Wind',
+      type: 'StartsUsing',
+      netRegex: { id: '8DF5', source: 'Apollyon', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Ihuykatumu Apollyon Swarming Locust First Cleaves',
+      type: 'Ability',
+      netRegex: { id: '8DF7', source: 'Apollyon', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get away after 3rd jump',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Apollyon Swarming Locust Second Cleaves',
+      type: 'Ability',
+      netRegex: { id: '8DF7', source: 'Apollyon', capture: false },
+      delaySeconds: 7.5,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get away after 3rd jump',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Apollyon Thunder III',
+      type: 'HeadMarker',
+      netRegex: { id: '006C' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Ihuykatumu Apollyon Wind Sickle',
+      type: 'Ability',
+      netRegex: { id: '8E05', source: 'Apollyon', capture: false },
+      durationSeconds: 8,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'In, then follow jump',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Apollyon Windwhistle 1',
+      type: 'Ability',
+      netRegex: { id: '8E07', source: 'Apollyon', capture: false },
+      delaySeconds: 7,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge line cleaves from whirlwind',
```

Comment:
This one I think might be slightly too much. Maybe "avoid whirlwind star lines"? It's still long, but the star is a known mechanic at this point in the game.

Reply:
> This one I think might be slightly too much. Maybe "avoid whirlwind star lines"? It's still long, but the star is a known mechanic at this point in the game.

Sure, that's fine.  Mostly, I felt a timed callout (of some kind) was a good idea, since there's no other visual indicator (that I could see anyway) that it's about to go off.